mountpoint-s3/tests/fuse_tests/statfs_test.rs

```diff
@@ -0,0 +1,82 @@
+use crate::common::fuse::{self, TestSessionCreator};
+use test_case::test_case;
+
+/// Tests that non-zero empty space is reported
+fn statfs_test_available_nonzero(creator_fn: impl TestSessionCreator, prefix: &str) {
+    let test_session = creator_fn(prefix, Default::default());
+    let mount_dir = test_session.mount_path();
+    let stats = nix::sys::statvfs::statvfs(mount_dir.into()).unwrap();
+    assert_ne!(stats.blocks_free(), 0);
+    assert_ne!(stats.blocks_available(), 0);
+    assert_ne!(stats.blocks(), 0);
+}
+
+/// Tests that default values from FUSER are reported for mpst fields
+fn statfs_test_fuser_defaults(creator_fn: impl TestSessionCreator, prefix: &str) {
+    let test_session = creator_fn(prefix, Default::default());
+    let mount_dir = test_session.mount_path();
+    let stats = nix::sys::statvfs::statvfs(mount_dir.into()).unwrap();
+    //assert_eq!(stats.name_max(), 255);
+    // These five aren't default values but set by us, so maybe drop
+    assert_eq!(stats.blocks(), u64::MAX / 1024);
+    assert_eq!(stats.blocks_free(), u64::MAX / 1024);
+    assert_eq!(stats.blocks_available(), u64::MAX / 1024);
+    assert_eq!(stats.files(), u64::MAX / 1024);
+    assert_eq!(stats.files_available(), u64::MAX / 1024);
```

Comment:
I'd drop these, but actually maybe we just combine with the test above. No need to separate them.

Reply:
Combined them with test above. Am now checking for > 0 (as that is what we really care about, not the actual number).